src/sentry/snuba/spans_rpc.py

```diff
@@ -298,6 +300,7 @@ def run_top_events_timeseries_query(
         limit,
         referrer,
         config,
+        None,
```

Comment:
Is this intentional set to none for the top events timeseries? And the sampling mode be implemented for any timeseries queries?